tests/Feature/Api/V1/MobileBootstrapTest.php

```
@@ -0,0 +1,172 @@
+    expect(Cache::has("user_dashboard_stats_{$user->id}"))->toBeTrue()
+        ->and(Cache::has("user_recent_reading_activity_series_{$user->id}"))->toBeTrue();
+});
+
+function createBootstrapReading(
+    User $user,
+    int $bookId,
+    string $dateRead,
+    string $createdAt,
+    int $chapter = 1,
+): ReadingLog {
+    return ReadingLog::factory()->for($user)->create([
```

PHPStan (2.2.7)

[error] Function createBootstrapReading() should return App\Models\ReadingLog but returns Illuminate\Database\Eloquent\Collection<int, Illuminate\Database\Eloquent\Model>|Illuminate\Database\Eloquent\Model.


(return.type)